crates/sdk/tests/integration_test.rs

```diff
@@ -373,7 +376,7 @@ fn test_e2e_proof_generation_and_verification() {
         )
         .unwrap();
     let evm_verifier = sdk
-        .generate_snark_verifier_contract(&params_reader, &agg_pk)
+        .generate_openvm_halo2_verifier_contract(&params_reader, &agg_pk)
```

Comment:
I think this test is actually duplicate / a subset of the new test you added; we don't need both to run in CI, and you can just delete this one